### ISSUE TITLE
Books to PV 

### ISSUE THREAD

**Issue by mdf-ido** [halkeye-helm-charts/ubooquity#2]
I'm trying to have the books directory create a PV just like the /config path. Any tips? I will share the changes as well in case anyone else needs it. 

I have been able to successfully create a separate mount path and after deployment I create a PV but I think permissions are getting in the way GUID/UID since I am not seeing the files. I'd like to understand how the PV for /config gets created so I can do the same for books or at least add another path with the correct permissions. 

Thanks!

**Comment by halkeye** [comment]
https://github.com/halkeye-helm-charts/ubooquity/blob/master/templates/pvc.yaml
https://github.com/halkeye/helmfile/blob/master/namespaces/media/ubooquity.yaml

I'm slowly going to start retiring my at home helm charts, moving to a single machine with docker compose, so I recommend forking or talking to @k8s-at-home group to add it to their charts

**Comment by mdf-ido** [comment]
Thank you! I will give that a shot. I was able to get the PVs to show up from NFS and was able to create folders and see them in the app but the app was not able to import the files even after using the same puid and guid. I went with Calibre + Calibre web and nfs-subdir-external-provisioner to share one PVC with ReadWriteMany and I'm ok with this for now. Will look into ubooquity later on. 


thanks again!
